main.go

```diff
@@ -38,7 +40,7 @@ func main() {
 		loggerConfig.IOWriter = os.Stdout
 		newLogger, err = micrologger.New(loggerConfig)
 		if err != nil {
-			panic(err)
+			panic(fmt.Sprintf("%#v", microerror.Maskf(err, "micrologger.New")))
```

Comment:
Would be cool do it like this: https://github.com/giantswarm/draughtsman-eventer/blob/fd040324c12b324e801e72a6e3cfda10611155d8/main.go#L31 as well.

Reply:
I think there it's even worse: https://github.com/giantswarm/draughtsman-eventer/blob/fd040324c12b324e801e72a6e3cfda10611155d8/main.go#L71

I can save I panic for crating a logger.

Reply:
I added mainWithError. 2 panic calls saved. It's something :)

Reply:
I'd like to get rid of this server factory at some point. Did you have some thoughts about it?

Reply:
My thinking is it will not go away. I cannot think of any way. It is also super unimportant. What would be nice would be to extend its interface to be able to return an error but this seems to be difficult as well and also super unimportant.